src/amo/sagas/reviews.js

```diff
@@ -297,7 +297,7 @@ function* deleteAddonReview({
       reviewId,
     });
 
-    yield put(unloadAddonReviews({ reviewId }));
+    yield put(unloadAddonReviews({ reviewId: isReplyToReviewId || reviewId }));
```

Comment:
Why does this switch the ID in the case of a reply? Can't you always just use `reviewId`?

There is a bug that you can reproduce like this:
- As a developer, submit a reply
- View your user profile page where you can see your reply
- Go back to the add-on listing and delete your reply
- View your user profile page again.

The reply you just deleted is still visible. It disappears if you reload the page, though.

<img width="625" alt="screenshot 2018-08-28 10 02 17" src="https://user-images.githubusercontent.com/55398/44731896-cec4d080-aaa9-11e8-8ef9-033f767d07de.png">

Reply:
I guess we need to clear both. I hadn't checked that scenario. We do need to clear based on `isReplyToReviewId` for a reply, otherwise we don't clear data for reviews which had this as a reply, but yes, we also need to clear data for the actual `reviewId` of the reply.